### DISCUSSION TITLE
Send current separate to Domoticz

### DISCUSSION THREAD

**Discussion by ElMyggo** [UtilitechAS/amsreader-firmware#400] [General]
Hi!
Found this great firmware because i buy module Pow-K!
I live in Sweden and has the Kamstrup 8660 meter with Rj-45 HAN-port
The Pow-K is on the way but i put the firmware on a D1 Mini Pro for little testing.
Is it possible to get the Current (L1,L2,L3) reading separate for sending to Domoticz?
today its combined 3 phase L1+L2+L3, i be interested in different iDx value/Phase
![fas](https://user-images.githubusercontent.com/111691693/212409474-1613fa54-542d-4497-ab69-31ad3065aefa.jpg)

**Comment by gskjold** [comment]
I personally have no experience with Domoticz, are there any documentation on the data format that should be used?

**Reply by ElMyggo** [reply]
Maybe there is no need for this, i use Domoticz and still waiting for my POW-K so i havent get any live data yet.
I get back when its up and running.

**Reply by ArnieO** [reply]
> Maybe there is no need for this, i use Domoticz and still waiting for my POW-K so i havent get any live data yet. I get back when its up and running.

Looking forward to that @ElMyggo !
